web3/main.py

```diff
@@ -23,16 +24,18 @@
 from web3.admin import Admin
 from web3.db import Db
 from web3.eth import Eth
+from web3.iban import Iban
```

Comment:
Side Note:  I'd like to extract the iban functionality into a stand-alone library and get it out of web3.  It's a format that isn't used or supported very broadly in the ecosystem and while I'm still fine supporting it, I'd rather it live in a different repository.